pilot/adapter/config/memory/monitor.go

```diff
@@ -49,6 +49,11 @@ type configstoreMonitor struct {
 
 // NewConfigStoreMonitor returns new Monitor implementation
 func NewConfigStoreMonitor(store model.ConfigStore) Monitor {
+	return NewConfigStoreMonitorWithBufferSize(store, BufferSize)
+}
+
+// NewConfigStoreMonitorWithBufferSize returns new Monitor implementation with the specified event buffer size
+func NewConfigStoreMonitorWithBufferSize(store model.ConfigStore, bufferSize int) Monitor {
```

Comment:
`NewBufferedConfigStoreMonitor`

Reply:
And in this case and above, I don't mind dropping the `New` either, so it's just `memory.BufferedConfigStoreMonitor(configStore)`. Your choice. We might also consider just calling it `(New)BufferedConfigMonitor` or even `(New)BufferedMonitor`

Reply:
Done. Renamed the other New method to be consistent.